tests/conftest.py

```diff
@@ -3,6 +3,13 @@
 
 from app import create_app
 
+FEATURE_REDIRECT_TO_FILE = True
+
+skip_if_direct_download = pytest.mark.skipif(
+    FEATURE_REDIRECT_TO_FILE,
+    reason="Redirecting directly to file"
+)
```

Comment:
Skip tests related to the landing page.

Reply:
Is there a situation where the skip will be `False`? As a developer, when would I switch this back manually to `False`?

Reply:
At the moment you see it's hardcoded to `true` because `FEATURE_REDIRECT_TO_FILE` is a constant. Basically this code is not very flexible but it should help devs in the future to know what to change/reintroduce easily helpful tests if landing pages make a come back in the future.